File: solutions/09-rafid/age of villagers/age of villagers/ageofvillagers/Triangle.cs
using System;

public class Triangle : Shape
{
    private int x1, y1, x2, y2, x3, y3;
    public Lines(int x1, int y1, int x2, int y2, int x3, int y3)
    {
        this.x1 = x1;
        this.x2 = x2;
        this.x3 = x3;
        this.y1 = y1;
        this.y2 = y2;
        this.y3 = y3;


    }
    public string Draw()
    {
        return "Drawing a triangle";
    }

}

File: solutions/40-arif-JAVA/AgeOfVillagers/src/shape/Triangle.java
package shape;

import javafx.scene.canvas.Canvas;
import javafx.scene.canvas.GraphicsContext;

public class Triangle implements IShape {
    double x,y;
    public Triangle(double x, double y) {
        this.x = x;
        this.y = y;
    }

    @Override
    public void draw(GraphicsContext graphicsContext) {

        new Line(x,y,x+10,y+10).draw(graphicsContext);
        new Line(x,y,x-10,y+10).draw(graphicsContext);
        new Line(x+10,y+10,x-10,y+10).draw(graphicsContext);



        System.out.println(x+" " +y);
    }
}
